rtrefresh/rt_refresh_manager.go

```diff
@@ -292,3 +293,17 @@ func (r *RtRefreshManager) runRefreshDHTQuery(key string) error {
 
 	return err
 }
+
+type loggableRawKeyString string
+
+func (lk loggableRawKeyString) String() string {
+	k := string(lk)
+
+	if len(k) == 0 {
+		return k
+	}
+
+	encStr := base32.RawStdEncoding.EncodeToString([]byte(k))
+
+	return encStr
+}
```

Comment:
It's a little unfortunate that this is here. It didn't seem worth it to make a logging subpackage, but I'm willing to budge here.